schema/schema_validator.py

```diff
@@ -129,8 +127,13 @@ def validate_test_data_with_schema(self):
 
         # Check for all the possible files
         json_file_pattern = os.path.join(self.test_data_base, '*', '*.json')
+        logging.info('JSON FILE_PATTERN: %s', json_file_pattern);
```

Comment:
for here and elsewhere (above & below): change log level to debug if this is not useful to see in a log file (ex: looking at a CI run)